app/api.py

```diff
@@ -75,6 +75,84 @@ def post_communityresource_register(body):
     return resource.to_dict(), 200
 
 
+def put_community_resource_edit(body):
+
+    geolocator = Nominatim() # will copy over from master branch
+    try:
+        _, (lat, lon) = geolocator.geocode(body["address"])
+    except expression as identifier:
+        return NoContent, 500
+
+    try:
+        CommunityResourceManager.edit_community_resource(int(body["number"]), body["name"], lat, lon, body["contact_name"], body["email"], body["phone_number"]) 
+    except NoExistingCommunityResource:
+        return NoContent, 500
+    except InvalidCommunityResourceInfo:
+        return NoContent, 500
+    return NoContent, 200
+
+
+def put_community_resource_edit_name(body):
```

Comment:
It's preferable to use a single put for a resource. You can choose to only update the fields included in the body for partial updates, or require the client to make a full get request first and change the values they want to update.

Reply:
I'll default to the latter option